src/python/dxpy/bindings/dxdataobject_functions.py

```diff
@@ -146,16 +146,32 @@ def get_handler(id_or_link, project=None):
 
 def describe(id_or_link, **kwargs):
     '''
-    :param id_or_link: String containing an object ID or dict containing a DXLink
+    :param id_or_link: String containing an object ID or dict containing a DXLink,
+                       or a list of object ID's or dict's containing a DXLink.
 
     Given an object ID, calls :meth:`~dxpy.bindings.DXDataObject.describe` on the object.
 
     Example::
 
         describe("file-1234")
     '''
-    handler = get_handler(id_or_link)
-    return handler.describe(**kwargs)
+    # If this is a list, extract the ids.
+    if isinstance(id_or_link, list):
```

Comment:
It would be nice to treat tuples, sets, or other sequence types in the same way you treat lists. What do you think about processing them in the opposite order...

```
if isinstance(id_or_link, basestring) or isinstance(id_or_link, dict):
    handler = get_handler(id_or_link)
    return handler.describe(**kwargs)
else:
    # id_or_link is now presumed to be a sequence of str/dict values...
    links = []
    # etc.
```